services/management/external.go

```diff
@@ -85,7 +85,7 @@ func (e ExternalService) AddExternal(ctx context.Context, req *managementpb.AddE
 		if req.MetricsMode == managementpb.MetricsMode_AUTO {
```

Comment:
While we are at that file, can we change the receives of those methods to pointer?

```diff
- func (e ExternalService) AddExternal
+ func (e *ExternalService) AddExternal
```